### ISSUE TITLE
The Python runtime is unzipped to the wrong path.

### ISSUE THREAD

**Issue by ntoll** [pybee/briefcase#143]
Using the latest release of Briefcase, the Python runtime is no longer in `Contents/Resources/python/bin` within the .app. Whatever tgunzips the support package plonks the sub-directories into `Resources`
rather than unzipping the `python` parent directory in there. Hence the path is wrong at the top of shell script in `Contents/MacOS` which actually launches the app.

**Comment by freakboy3742** [comment]
@ntoll Thanks for the report. I've taken a look, and haven't been able to reproduce the problem you're seeing. Can you tell me
* The version of briefcase you're using (Current is 0.2.4)
* The version of Python you're using (I tested with 3.4, 3.5 and 3.6)
* The support package that was identified during the install (On 3.5, it used https://pybee-briefcase-support.s3-us-west-2.amazonaws.com/Python-Apple-support/3.5/macOS/Python-3.5-macOS-support.b8.tar.gz)
* A log of the installation process.

I think you mentioned you were using a custom support package; that package is designed to be unzipped *in * the Resources directory. If this is the case, can you confirm the top level directory structure of that custom support package? The "stock" support packages contain a single `python` directory, as well as a `VERSIONS` file in the root. The `VERSIONS` file is necessary to prevent Briefcase from collapsing the directory structure.

**Comment by ntoll** [comment]
Weird... here's the travis build with all the details: https://travis-ci.org/mu-editor/mu/jobs/391736987

The custom version of Python is the most recent release from here: https://github.com/mu-editor/mu_portable_python_macos

If you can't reproduce then my guess it's PEBCAK... ;-) (If so, what am I doing wrong?).

**Comment by freakboy3742** [comment]
Ok - I can now reproduce the problem, and it's not PEBCAK; it's what I suggested in my original response.

The support package you have published [here](https://github.com/mu-editor/mu_portable_python_macos/releases) contains a full Python install, including the python directory, ready to be extracted into the `Resources` folder. However, there's nothing in the tarball that *isn't* in the `python` directory, so the unzipper that briefcase uses tries to be smart and collapses the python directory out of the way - resulting in the `bin`/`lib`/`etc` directories being placed directly into the `Resources` folder.

If you rebuild that support package and put a single file in the root of the tarball, the problem will be resolved. I use a `VERSIONS` file so I can track exactly which support package was used to track the binaries, then the `python` directory will be preserved; but it could be any file you want. You just need to provide a reason for briefcase to *not* collapse the top level `python` directory.

i.e., `tar ztvf mu_support_python.tar.gz` should return:

    VERSIONS
    python/
    python/bin
    python/bin/python3
    python/bin/python3.6
    ...


The other thing you'll need to do is put in an alias for python3 - your support package produces `python/bin/python3.6`, but not `python/bin/python3`, and it's the latter which the project template expects to find at runtime. 

One last tip/suggestion - Briefcase originally hosted it's support files on Github; however, we stopped doing that after our first sprint. It turns out it's really easy to hit Github's rate limit in a sprint situation if you have a people downloading large support files. This might not be a problem in your particular case, but I thought I'd flag it just in case.
